### PR TITLE
Portfolio: Setup Portfolio

### PR DESCRIPTION
**Pull request by samiullahbahadur**
merge to main

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+               
+               
+                <!-- <li><a href="# ">Portfolio</a></li>
+                <li><a href="# ">About</a></li>
+                <li><a href="# ">Contact Us</a></li>
+                <li><a href="# "><i class="fas fa-envelope"></i></a></li> -->
+                <li > <i class="fas fa-bars fa-2x" ></i></li>
+               
+            </ul> 
+        </div>
```
Comment (Amrhub) [inline index.html]: - [ ] Let's make sure we use `nav` tag instead of div for better HTML Semantic

File: index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                <h1>Hey there. <br>I am ُSamiullah Bahadur</h1>
+                <h2>I am a Software Developer</h2>
+                <p>I can help you build a product , feature or website Look 
+                    through some of my work and experience! If you like what
+                     you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+        </div>
+        <div class="landing-image ">
+            <img  src="image/landing-origin.png" alt="landing-image "> 
+        </div>
```
Comment (Amrhub) [inline index.html]: - [ ] Let's make sure we use any image for styling purposes as a background image  in CSS we should only use images where we need it as part of content, Also, let's make sure we use the same background image as used in Figma 
image

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @samiullahbahadur,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
✔️ All linters are passing
✔️ Documentated your project in a professional way
✔️ Clean code and easy to read

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            <nav class="navbar ">
+                <div class="humberger-menu">
+                    <a href="# "><i class="fas fa-bars fa-2x"></i></a>
+                </div>
+                <!-- <ul>
+                    <li><a href="# ">Portfolio</a></li>
+                    <li><a href="# ">About</a></li>
+                    <li><a href="# ">Contact Us</a></li>
+                    <li><a href="# "><i class="fas fa-envelope"></i></a></li>
+                </ul> -->
```
Comment (andersonlebon) [inline index.html:33]: ## [OPTIONAL]
- [ ] To make this file more clear and clean I would suggest you remove all commented code.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Samiullah Bahadur Portfolio
+
+> A simple Portfolio project with about Samiullah Bahadur
+
+You can download my project  copy this link(git@github.com:samiullahbahadur/Portfolio.git) and git clone paste the link in the terminal
+
```
Comment (andersonlebon) [inline README.md:6]: ## [OPTIONAL]
- [ ] To make your `README` file look more professional. I would kindly suggest you add a screenshot of the first section of this project just only to give visitors a short review of how your project looks like. :clap:

**Review by andersonlebon — APPROVED** [review]
## Hi @samiullahbahadur 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions

- [ ] Kindly I would recommend you add a good pull request description
image


- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**